pyproject.toml

```diff
@@ -102,7 +101,7 @@ pre-commit = "^3.5.0"
 lxml = "^5.0.0"
 uv = "^0.1.5"
 toml = "^0.10.2"
-werkzeug = "^2.2.0" # Added as mitmproxy8 -> flask<2.1
+werkzeug = "^3.0.3"
```

Comment:
looking at the comment we had before, is `werkzeug` still needed as a dependency? Or can we do without it?